### ISSUE TITLE
Suspicious cc attachment

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#555]
Ordinarily the cc marks a conjunct attaching via `conj`, or it may be sentence-initial (in which case the head probably attaches to root). But not always:

Queries:

- [x] [GUM](https://universal.grew.fr/?custom=675a52b543798)
- [x] [EWT](https://universal.grew.fr/?custom=675a52c2a40d4)

If there is a coordinating conjunction, I assume we should prefer a conj analysis over alternatives like parataxis or advcl? Some patterns that show up as non-conj:

- 10 days or so
- SENTENCE, and so on.
- SENTENCE, but not when...
- SENTENCE, but only after V-ing.

https://universaldependencies.org/u/dep/parataxis.html mentions "non-conjunction connective[s]", which suggests that clauses connected by conjunctions are conj instead. Also, conj seems to be in the majority when there is a parenthesis: https://universal.grew.fr/?custom=675a54dfb871c

**Comment by nschneid** [comment]
- [ ] [Constraints on cc parent function](https://universal.grew.fr/?custom=67743c8637083) implemented in GUM validator amir-zeldes/gum@efab974 (and I will add to neaten.py)

@amir-zeldes do you agree with the above that conj should be preferred for "and so on" etc.?

**Comment by amir-zeldes** [comment]
Yes, definitely!

**Comment by nschneid** [comment]
Which should be the head of "so on"—"so" or "on"?

**Comment by amir-zeldes** [comment]
"on"

**Comment by nschneid** [comment]
See #1087

**Comment by nschneid** [comment]
"This guy is, **and pardon my French**, a jerk." `parataxis`? We can't have left-pointing `conj`. Unless you think it's `reparandum`...it strikes me as a clean parenthetical, not a disfluency.

**Comment by amir-zeldes** [comment]
Yes, parataxis + cc. Like root, parataxis can head a complete utterance, so it makes sense it can take a cc child too.

**Comment by nschneid** [comment]
OK. I am also seeing ccomp and acl cases due to quotation, where the quote begins with a CCONJ.

**Comment by nschneid** [comment]
Now I see that the validation rule already allows `ccomp`. Should we add `acl` for:

> Work it into your speech-- something along the line of, "And Jen, you come from great people and have fantastic friends.

**Comment by amir-zeldes** [comment]
That's certainly a valid example, but it's metalinguistic, and anything is possible there, so I tend to just list such cases as allowed exceptions, rather than allow it wholesale.
